ci/deploy/gather-indev/cli.ts

```diff
@@ -0,0 +1,32 @@
+import { Octokit } from '@octokit/rest';
```

Comment:
RMB -> Format Document in VS Code

Reply:
Hmm.. I've actually did not indent function body on purpose, because I feel like that's kinda upper scope anyway, it's needed only to allow using `await`. But ok, reformatted, that's kinda arguable choice (though many C++ codeguides do not indent namespace contents for example)

Reply:
Or were you talking about `'` instead of `"`?